### PR TITLE
feat:add get holders by atomical_id

### PR DESCRIPTION
**Pull request by shadowv0vshadow**
Add a new api `blockchain.atomicals.get_atomical_holder`, it can receive three params. atomical_id, limit(default 100), offset(default 0). for example, if you want get 'avm' token holders. you can send by following curl command.

```
curl -X POST http://0.0.0.0:8080/proxy/blockchain.atomicals.get_atomical_holder -d '{"params":["d4c48c4db504282e987776cba26f78fbc1573460a02c45d9e5de37c5b13ef751i0", 3, 0]}' -H "Content-Type: application/json" | jq
```
will return:
```
{
  "success": true,
  "response": [
    {
      "percent": 0.01019047619047619,
      "address": "bc1pf70gkj3zcxqmq4v3cjjngnpalhajnsc8fld9wl9xq547pvqgqmmqjw6j2f",
      "holding": 214000
    },
    {
      "percent": 0.008952380952380953,
      "address": "bc1px95q3n0e39np6slcdxyy5rjr6erm54zk340wq45fxh5sg7skzq8q804wpj",
      "holding": 188000
    },
    {
      "percent": 0.008714285714285714,
      "address": "bc1p5ue8euvu74em4vdlytam25z2ypxzd7vnh9w6f227da5kzxmx66dqdqrvx6",
      "holding": 183000
    }
  ]
}
```

Use this api  should  install bitcointx pythonlib `pip3 install python-bitcointx`.
This lib depend on secp256k1. run `brew install secp256k1` on macOs, and run `apt-get install libsecp256k1-0`  on ubuntu.

### CODE REVIEW

**Comment by atomicals** [comment]
Fantastic feature! Can you please update readme.md and also `requirements.txt` for the dependency?

**Comment by shadowv0vshadow** [comment]
I remove the python-bitcointx lib,  replaced it with `electrumx/lib/segwit_addr.py ` so `requirements.txt` and readme.md no need to update now

File: electrumx/server/db.py [hunk 1]
```diff
@@ -1386,6 +1386,44 @@ def query_location():
...
+            for atomical_active_location_key, atomical_active_location_value in self.utxo_db.iterator(prefix=atomical_active_location_key_prefix):
+                if atomical_active_location_value:
+                    location = atomical_active_location_key[1 + ATOMICAL_ID_LEN : 1 + ATOMICAL_ID_LEN + ATOMICAL_ID_LEN]
+                    atomical_output_script_key = b'po' + location
+                    atomical_output_script_value = self.utxo_db.get(atomical_output_script_key)
+                    location_script = atomical_output_script_value
+                    location_value, = unpack_le_uint64(atomical_active_location_value[HASHX_LEN + SCRIPTHASH_LEN : HASHX_LEN + SCRIPTHASH_LEN + 8])
+                    
+                    script = location_script.hex()
+                    # TODO
```
Comment (atomicals) [inline electrumx/server/db.py:1406]: How is that possible ?

**Comment by atomicals** [comment]
@shadowv0vshadow Merged, but there seems to be a problem with the `addr` display. Can you check and fix in another PR please? Example of problem (Notice the renaming of the method to just `get_holders`)

`https://ep.atomicals.xyz/proxy/blockchain.atomicals.get_holders?pretty&params=[... your params ]`

**Comment by shadowv0vshadow** [comment]
> @shadowv0vshadow Merged, but there seems to be a problem with the `addr` display. Can you check and fix in another PR please? Example of problem (Notice the renaming of the method to just `get_holders`)
> 
> `https://ep.atomicals.xyz/proxy/blockchain.atomicals.get_holders?pretty&params=[... your params ]`
> 
> ```
> {
>   "success": true,
>   "response": [
>     {
>       "percent": 0.017857142857142856,
>       "address": "",
>       "holding": 375000
>     },
>     {
>       "percent": 0.009523809523809525,
>       "address": "",
>       "holding": 200000
>     } 
> ```

@atomicals #45 fixed

**Pull request merged**